config/locales/en.yml

```diff
@@ -3,3 +3,6 @@
 
 en:
   hello: "Hello world"
+  spree:
+    add_new_codes: 'ADD NEW CODES'
```

Comment:
Let's use upcase/css where translation is being used and write here in regular case.